lib/components/viewers/RouteRow.js

```diff
@@ -154,7 +150,8 @@ export class RouteRow extends PureComponent {
           className="clear-button-formatting"
           isActive={isActive}
           onClick={this._onClick}
-          onMouseEnter={this._onMouseEnter}
+          onFocus={this._onFocusOrMouseEnter}
```

Comment:
Realizing now we need touch support as well! Do you think it makes sense to include a `onTouchStart` handler as well?

Reply:
I added a `touchStart` handler in https://github.com/opentripplanner/otp-react-redux/pull/675/commits/13adba9ee416c7cb0a9db96b7b16f754c8a9632d, but I am really wondering what the expanded `RouteRow` is really trying to achieve.

Reply:
The geometry data is quite massive... But agreed we should think about figuring out how to load it in at clever times